src/models/user.ts

```diff
@@ -0,0 +1,50 @@
+import mongoose from "mongoose";
+
+export interface IUser {
```

Comment:
Same here as with the appointment. Add an id as a string generated by [uuid](https://www.npmjs.com/package/uuid)